bin/print.js

```diff
@@ -1,18 +1,19 @@
-#!./node_modules/.bin/ts-node
+#!/usr/bin/env node
 
 import fs from "fs";
 import prettier from "prettier";
 
-import plugin from "../src/plugin";
+import plugin from "../src/plugin.js";
 
 const code = fs.existsSync(process.argv[2])
   ? fs.readFileSync(process.argv[2], "utf-8")
   : process.argv.slice(2).join(" ").replace(/\\n/g, "\n");
 
 const options = {
   parser: "xml",
-  plugins: [plugin as any as string],
-  xmlWhitespaceSensitivity: "ignore"
+  plugins: [plugin],
+  xmlWhitespaceSensitivity: "ignore",
+  embeddedLanguageFormatting: "auto"
 };
 
 console.log(prettier.format(code, options));
```

Comment:
Forgot to `await` ?

Reply:
Ahh yeah I don't run this script often. That's fixed up now. Thanks!